src/goto-analyzer/static_analyzer.cpp

```diff
@@ -1,238 +0,0 @@
-/*******************************************************************\
-
-Module:
-
-Author: Daniel Kroening, kroening@kroening.com
-
-\*******************************************************************/
-
-#define USE_DEPRECATED_STATIC_ANALYZER_H
-#include "static_analyzer.h"
-#undef USE_DEPRECATED_STATIC_ANALYZER_H
-
-#include <fstream>
-
-#include <util/threeval.h>
-#include <util/json.h>
-#include <util/xml.h>
-
-#include <analyses/interval_domain.h>
-
-class static_analyzert:public messaget
```

Comment:
This class should really be removed. I think, however, that it is being used by @smowton?